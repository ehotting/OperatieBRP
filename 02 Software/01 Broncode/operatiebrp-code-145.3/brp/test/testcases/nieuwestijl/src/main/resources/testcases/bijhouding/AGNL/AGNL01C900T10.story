Meta:
@auteur                 tjlee
@status                 Klaar
@regels                 R2176
@sleutelwoorden         voltrekkingHuwelijkInNederland,registratieNaamgebruik,TjieWah,AGNL01C900T10
@usecase                UCS-BY.HG

Narrative:
R2176 Huwelijk en geregistreerd partnerschap, voltrekking huwelijk in Nederland, actie registratieAanvangHuwelijkGeregistreerdPartnerschap, registratieNaamgebruik

Scenario: R2176 Persoon.Naamgebruik is niet ingevuld.
          LT: AGNL01C900T10

Gemeente BRP 1

Given alle personen zijn verwijderd

Given enkel initiele vulling uit bestand /LO3PL-AGNL/Marjan.xls
Given enkel initiele vulling uit bestand /LO3PL-AGNL/Victor.xls

When voer een bijhouding uit AGNL01C900T10.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/AGNL/expected/AGNL01C900T10.txt

