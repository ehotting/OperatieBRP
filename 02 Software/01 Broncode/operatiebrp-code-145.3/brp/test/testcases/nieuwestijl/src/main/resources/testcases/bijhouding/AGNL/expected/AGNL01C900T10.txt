cvc-complex-type.2.4.a: Invalid content was found starting with element 'brp:indicatieAfgeleid'. One of '{"http://www.bzk.nl/brp/brp0200":code}' is expected.